### DISCUSSION TITLE
A problem about installation failure

### DISCUSSION THREAD

**Discussion by ShadowXTz** [Security-Onion-Solutions/securityonion#8957] [Unsupported Versions]
It took me six or seven hours to install the security onion, but the installation process showed that the installation failed. After that, I chose to restart. After restart, I used "sudo so allow" to run and input the email account password I set during installation when accessing on my physical machine. However, "The provided credentials are invalid. Please try again" is displayed on the web interface Is there a way to reset the mailbox account password after installation? I installed it on a virtual machine with 4GB memory and 200GB disk space.

**Comment by dougburks** [comment]
It shouldn't take six or seven hours to install Security Onion. If it showed that the installation failed, then there's no guarantee that anything will work properly.

We'd recommend trying a fresh installation with the latest ISO image. For best results, install on a dedicated machine as recommended by best practices:
https://docs.securityonion.net/en/2.3/best-practices.html

If you must use a VM, increase the resources assigned to that VM if possible:
https://docs.securityonion.net/en/2.3/hardware.html

If you can only allocate 4GB RAM to the VM, make sure that you are choosing IMPORT mode as shown in the first time users guide:
https://docs.securityonion.net/en/2.3/first-time-users.html.
